src/components/GeoMap.js

```diff
@@ -42,16 +42,10 @@ export default class GeoMap {
     this.node.setAttribute('id', 'mt-map');
     mapWrapper.appendChild(this.node);
 
-    this.svg = d3.select(this.node)
-      .append('svg')
-      .attr('id', 'mt-map-svg')
-      .attr('xmlns', 'http://www.w3.org/2000/svg')
-      .attr('xmlns:xlink', 'http://www.w3.org/1999/xlink')
-      .attr('viewBox', `0 0 ${this.getWidth()} ${this.getHeight()}`)
-      .attr('width', this.getWidth())
-      .attr('height', this.getHeight());
+    this.svg = d3.select(this.node).append('svg').attr('id', 'mt-map-svg').attr('xmlns', 'http://www.w3.org/2000/svg').attr('xmlns:xlink', 'http://www.w3.org/1999/xlink').attr('viewBox', `0 0 ${this.getWidth()} ${this.getHeight()}`).attr('width', this.getWidth()).attr('height', this.getHeight());
```

Comment:
Are you using any linter to make these style changes?

Reply:
Oh, I was using prettier for code formatting locally, disabled it now for this project, should be fine now.

Reply:
Done @melalj